modules/ROOT/partials/individually-licensed-components/import-from-word-and-export-to-word/import-from-word-and-export-to-word-api-usage-on-premises.adoc

```diff
@@ -0,0 +1,14 @@
+[[api-usage]]
+== API usage
+
+{pluginname} On-Premises consists of two services:
+
+* xref:exportword.adoc[Export to Word] provides conversion from an HTML document to a `.docx` file via Restful API.
+* xref:importword.adoc[Import from Word] allows importing `.docx` file and converting it into a styled HTML document with comments and suggestions attached.
+
+The API is available on `http://localhost:[port]` (by default the port is 8080).
+
+> The REST API documentation is available at `http://localhost:[port]/docs`.
```

Comment:
The URL does not render properly on the site. It should read `http://localhost:[port]/docs`.